File: backup_before_industrialisation_20250626/backend/metier_generator/templates/threed/tests_centralises/services/test_services.js
// Tests avancés pour les services threed
const { runService, getServiceStatus } = require('../../api/services');
describe('Services threed', () => {
  it('doit exécuter le service de notification sans erreur', async () => {
    const result = await runService('notification', { message: 'test' });
    expect(result.success).toBe(true);
  });
  it('doit retourner le statut du service de monitoring', () => {
    const status = getServiceStatus('monitoring');
    expect(['running', 'stopped']).toContain(status);
  });
});

File: backend/metier_generator/templates/threed/tests_centralises/services/test_services.py
"""
Tests centralisés pour les services threed (Python)
"""

from ...services import run_service, get_service_status


def test_run_notification_service():
    result = run_service("notification", {"message": "test"})
    assert result["success"] is True


def test_monitoring_service_status():
    status = get_service_status("monitoring")
    assert status in ["running", "stopped"]
